pymc3/plots/__init__.py

```diff
@@ -53,6 +52,38 @@ def wrapped(*args, **kwargs):
 densityplot = map_args(az.plot_density)
 pairplot = map_args(az.plot_pair)
 
+# addition arg mapping for compare plot
+swaps = [
+    ('varnames', 'var_names')
+]
+@functools.wraps(az.plot_compare)
+def compareplot(*args, **kwargs):
+    args = list(args)
+    comp_df = args[0].copy()
+    if 'WAIC' in comp_df.columns:
+        comp_df = comp_df.rename(index=str,
+                                 columns={'WAIC': 'waic',
+                                          'pWAIC': 'p_waic',
+                                          'dWAIC': 'd_waic',
+                                          'SE': 'se',
+                                          'dSE': 'dse',
+                                          'var_warn': 'warning'})
+    elif 'LOO' in comp_df.columns:
+        comp_df = comp_df.rename(index=str,
+                                 columns={'LOO': 'loo',
+                                          'pLOO': 'p_loo',
+                                          'dLOO': 'd_loo',
+                                          'SE': 'se',
+                                          'dSE': 'dse',
+                                          'shape_warn': 'warning'})
+    args[0] = comp_df
+    args = tuple(args)
```

Comment:
Why does args need to be coerced into a tuple? The list can be unpacked in line 85 as well right?